src/translib/transformer/xfmr_intf.go

```diff
@@ -1298,6 +1298,10 @@ func convertIpMapToOC (intfIpMap map[string]db.Value, ifInfo *ocbinds.Openconfig
 
     subIntf = ifInfo.Subinterfaces.Subinterface[0]
     ygot.BuildEmptyTree(subIntf)
+    ygot.BuildEmptyTree(subIntf.Ipv4)
```

Comment:
The ygot.BuildEmptyTree() initializes all the subcontainers of a particular point in the YANG tree, i.e. starting at the subIntf provided. Hence, I think these changes are not required. I am wondering how the changes resolve the issue.

Reply:
> The ygot.BuildEmptyTree() initializes all the subcontainers of a particular point in the YANG tree, i.e. starting at the subIntf provided. Hence, I think these changes are not required. I am wondering how the changes resolve the issue.

Reply:
If the call is made at Ip4v4 and ipv6 level calling build tree at subinterface level doesn't help building as the child is not nil. This method should handle subinterfaces, subinterfaces/subinterface, subinterfaces/subinterface/Ipv4, subinterfaces/subinterface/Ipv6,  subinterfaces/subinterface/Ipv4/addresses, subinterfaces/subinterface/Ipv6/addresses, subinterfaces/subinterface/Ipv4/addresses/address and  subinterfaces/subinterface/Ipv6/addresses, subinterfaces/subinterface/Ipv6/addresses/address  URI's. As buildemptytree is no op if children are not nil, we can avoid checks and call build empty tree at every level we support.